tests/test_utils.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import torch as torch
```

Comment:
Any reason for this over just `import torch`?

Reply:
Ah haha, clearly just my brain on autopilot.